include/mata/nfa/nfa.hh

```diff
@@ -389,6 +394,15 @@ public:
      * @return True if some new transition (and sink state) was added to the automaton.
      */
     bool make_complete(const Alphabet& alphabet) { return this->make_complete(alphabet, this->num_of_states()); }
+
+    /**
+     * Complement deterministic NFA in-place by adding a sink state and swapping final and non-final states.
```

Comment:
"deterministic NFA" sounds a bit funny, maybe just DFA? Or deterministic automaton.

Reply:
I was thinking about it as well. I wanted to emphasize the `deterministic` part, which a reader might ignore when using just `DFA`. Therefore, I will use `deterministic automaton` instead.